product_docs/docs/pgd/4/bdr/index.mdx

```diff
@@ -26,8 +25,8 @@ navigation:
   - twophase
   - catalogs
   - functions
-title: BDR (Bi-Directional Replication)
-
+redirects:
+  - /pgd/4/overview/bdr
```

Comment:
Instead update this rule: https://github.com/EnterpriseDB/docs/blob/develop/static/_redirects#L40